test/python/transpiler/test_preset_passmanagers.py

```diff
@@ -629,22 +629,22 @@ def test_layout_tokyo_fully_connected_cx(self, level):
         }
 
         dense_layout = {
-            2: qr[0],
```

Comment:
What makes this end up different? Presumably it's still deterministic?

Reply:
It's actually caused by a bug in the python implementation before this. The noise properties were never actually populated in the error matrix for 2q errors as there was a type mismatch. It was trying to do `[0, 1] == (0, 1)` to see if an edge in the coupling map matched an edge in the noise properties, which was evaluating as False because of list vs tuple. I caught that and fixed it as part of this, so this is the change in result due to actually using the cx error rate for the device to pick the best dense layout. You can look at https://github.com/Qiskit/qiskit-terra/pull/7740/commits/7537709a9306aa5ffb17404a75dba7f4d47b3599 where I fixed this and I had to the update the test after making that change in: https://github.com/Qiskit/qiskit-terra/pull/7740/commits/34aeb1383d9488d942182c16d008d889dbb07a36 

This should always be deterministic for this test now (since the noise properties for the `FakeTokyo` backend are fixed).